src/test/llmnode_test.cpp

```diff
@@ -875,14 +875,47 @@ TEST_F(LLMFlowHttpTest, unaryCompletionsStopStringEmpty) {
         ovms::StatusCode::OK);
 }
 
+TEST_F(LLMFlowHttpTest, streamBeamSearchCompletions) {
+    std::string requestBody = R"(
+        {
+            "model": "llmDummyKFS",
+            "stream": true,
+            "best_of": 2,
+            "prompt": "What is OpenVINO?"
+        }
+    )";
+
+    ASSERT_EQ(
+        handler->dispatchToProcessor(endpointCompletions, requestBody, &response, comp, responseComponents, &writer),
+        ovms::StatusCode::PARTIAL_END);
+}
+
+TEST_F(LLMFlowHttpTest, streamBeamSearchChatCompletions) {
+    std::string requestBody = R"(
+        {
+            "model": "llmDummyKFS",
+            "stream": true,
+            "best_of": 2,
+            "messages": [
+            {
+                "role": "user",
+                "content": "What is OpenVINO?"
+            }
+            ]
+        }
+    )";
+
+    ASSERT_EQ(
+        handler->dispatchToProcessor(endpointChatCompletions, requestBody, &response, comp, responseComponents, &writer),
+        ovms::StatusCode::PARTIAL_END);
+}
```

Comment:
Those assertions do not look good. Partial end is a valid status for streaming processing and with latest changes we reject it. See `TEST_F(LLMFlowHttpTest, unaryChatCompletionsStreamOptionsSetFail)` where we also have bad parameters in the request. 

Also please add `...Fail` suffix to the test name to indicate it's a negative test.